sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -12,15 +14,13 @@ public class RecognizeHealthcareEntitiesTests : TextAnalyticsClientLiveTestBase
     {
         public RecognizeHealthcareEntitiesTests(bool isAsync) : base(isAsync) { }
```

Comment:
I took advantage and updated the test to be closer to the other ones we have in the TA library